cmd/deploy.go

```diff
@@ -21,24 +23,34 @@ var deployCmd = &cobra.Command{ //nolint:gochecknoglobals
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		projectId := flags.GetProjectId()
+		if projectId == "" {
```

Comment:
There's a check inside the API client but we should avoid all the extra work early on